.travis.yml

```diff
@@ -109,3 +117,4 @@ jobs:
       after_success:
         - # npx nyc report --reporter=lcov | npx codecov
```

Comment:
Why is this line still here, did we use to have codecov enabled before?
And why should we do it differently now than we did before?

Reply:
That's something I missed :open_mouth:

Reply:
I followed up using npm. Can be done this way too